tests/test_rules.py

```diff
@@ -1125,6 +1125,9 @@ def test_field_name_typo(self):
                 typos_ = typos + [("Targetobject", "TargetObject"), ("Eventtype", "EventType"), ("Newname", "NewName")]
             elif "process_creation" in logsource:
                 typos_ = typos + [("Parentimage", "ParentImage"), ("Integritylevel", "IntegrityLevel"), ("IntegritiLevel", "IntegrityLevel")]
+            elif "file_access" in logsource:
+                del(typos[1]) # We remove the entry to "TargetFileName" to avoid confusion
```

Comment:
```suggestion
                del(typos[typos.index(("TargetFileName", "TargetFilename"))]) # We remove the entry to "TargetFileName" to avoid confusion
```